### PR TITLE
style(frontend): Update ExtrenalLink hover state color [OISY-62]

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

The component `ExternalLink` should be updated with the new color pallets when hovered/active.

### CODE REVIEW

File: src/frontend/src/lib/components/ui/ExternalLink.svelte [hunk 1]
```diff
@@ -20,6 +20,8 @@
 	class:text-primary={color === 'blue'}
 	class:hover:text-inherit={color === 'blue'}
 	class:active:text-inherit={color === 'blue'}
+	class:hover:text-blue-ribbon={color === 'inherit'}
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/ui/ExternalLink.svelte:23]: it's a bit weird no? Blue is inherit but inherit is text-blue-ribbon.

Should we rename the value 'inherit'?

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/ui/ExternalLink.svelte:23]: well, those are the `hover`/`active` classes right? The idea is that `blue`/`inherit` refers to the main color of the text in the normal state

Comment (peterpeterparker) [reply src/frontend/src/lib/components/ui/ExternalLink.svelte:23]: I would honestly expect `class:hover:text-inherit={color === 'inherit'}`

Comment (peterpeterparker) [reply src/frontend/src/lib/components/ui/ExternalLink.svelte:23]: Ah now I understand. Yeah it's a bit confusing.

Comment (peterpeterparker) [reply src/frontend/src/lib/components/ui/ExternalLink.svelte:23]: The color is the idle state and the class is the active state.

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

**Pull request merged**